packages/react-native-web/src/exports/View/index.js

```diff
@@ -39,9 +39,40 @@ const forwardPropsList = {
   pointerEvents: true
 };
 
+type RNW$MeasureLayoutCallback = (
```

Comment:
Why do these types all have a RNW prefix?

Reply:
Removed prefix and moved around to match RNW style more than Twitter style